### ISSUE TITLE
I'm not sure if it's working correctly

### ISSUE THREAD

**Issue by acutbal** [microg/IchnaeaNlpBackend#11]
Hi!! :smiley: 

I've installed Networklocation.apk and IchnaeaNlpBackend in my Meizu MX3 as you explain in the main Readme doc. But I'm not sure if it works correctly, let me explain, I hope you will understand me because English it's not my language. Apologizes…

First of all, before installing the apk and the backend I was sure that the phone couldn't locate me by wifi/3G. Then installed Networklocation.apk plus IchnaeaNlpBackend and started the testings. What I did:

1.- In my office the phone located me correctly by wifi. Then, I turned off wifi and turned on the 3G connection. I'd checked the position and I was still in the same place.
2.- After, when I arrived at home I connected the phone to my wifi, I live in another city 20 kms far from my work. I refreshed some apps which use the location service and still appeared to be in work. 
3.- Then I installed AppleWifiNlpBackend, checked again and then the apps showed me the correct location, at home.
4.- Then, I selected the IchnaeaNlpBackend again,  I turned off wifi and turned on data. The location still was on my work, 20 kms far from home.
5.- After, I did the reverse process, turned off data and turned on wifi.  IchnaeaNlpBackend located me at the right place.

And now, I really have no idea if it works or not, I'm a non-technical user so  I'm really lost… When the backend connects to Mozilla servers? Every determined period of time?

Thank you very much and kind regards!! :smile:

**Comment by mar-v-in** [comment]
The problem here is likely that there is no or incomplete information about cell towers at your location. Thus the IchnaeaNlpBackend relies on the wi-fi information. If you disable wi-fi this is no longer possible and the location can't be updated. Some phones also report cell information. Please check the cell database coverage at http://gsm-map.com/

**Comment by acutbal** [comment]
Thank you very much for your help!!
I understand, perhaps  there isn't enough data on my area, I should check the data available in Tarragona.

One more question, can I use two different backends at the same time, one for Wi-Fi and another one for gsm?

Thank you very much and very best regards!! :D

**Comment by mar-v-in** [comment]
> One more question, can I use two different backends at the same time, one for Wi-Fi and another one for gsm?

Yes that's possible and an intended use. Select as many backends as you like.

**Comment by acutbal** [comment]
Thank you very much for your help!!

I'll try, I hope I can run it.

Again, thank you very much and very best regards!! :D

**Comment by sneakybidder** [comment]
I want to hide/kill the location app???? S3 Sph-L710  4.4.2. I just like ur last commenter. I rooted and I'm putting  a rom on. But so many apps hide/chg ur location. Which should i use? How? If there are files xactly names are necessary.  Thank you.
-jayc

**Comment by xuhdev** [comment]
@mar-v-in The URL you gave (`http://gsm-map.com`) has actually been broken. Can you fix it, please? Thanks!
